packages/lab/src/pill-next/PillNext.css

```diff
@@ -31,6 +30,11 @@
   font-size: var(--salt-text-fontSize);
   font-weight: var(--salt-text-fontWeight);
   line-height: var(--salt-text-label-lineHeight);
+  min-width: 4em;
```

Comment:
is this design driven?

Reply:
Good spot!! 
i just brought it a bit lower in the stylesheet because i was originally thinking of having both button's together given how much they share, but this should probably come from the theme. 
@yunjungyeh let's have a chat about this.

Reply:
@alycrys mystery solved, it is meant to be `4 * var(--salt-text-fontSize)` (which is basically the same as 4em) with pain in my heart I'll change it, as it shows intention better, oh good old relative units...

Reply:
@joshwooding

Reply:
All form controls have 4em as their min width so I assume this is to match that